src/actions/playersActions.js

```diff
@@ -12,10 +18,23 @@ type Action =
   | UpdatePlayersListAction;
 
 
-function updatePlayersList(): Action {
+function updatePlayersList(id): Action {
   return {
     type: UPDATE_PLAYERS_LIST
-  }
+  };
+}
+
+function initializePlayersList(total): Action {
+  const createdUsers = generateUsers(total);
+  const users = createdUsers.reduce((obj, currItem) => {
```

Comment:
obj as a variable name could be more descriptive, I would suggest for maps/objects to use the naming pattern valueByKey or keyToValue so in this case it looks like it would be usersByID or idToUsers

Reply:
Thanks, this is a great suggestion and I will adopt. I always have problems with naming these variables in map/objects.